client/galaxy/scripts/apps/analysis.js

```diff
@@ -174,6 +176,10 @@ window.app = function app( options, bootstrapped ){
             this.page.display( new CustomBuilds.View() );
         },
 
+        show_dataset_edit_attributes : function() {
+            this.page.display( new DatasetEditAttributes.View( Utils.getQueryString( 'dataset_id' ) ) );
```

Comment:
I noticed that we use something like `Galaxy.params.dataset_id` in other locations. It might be better to use that here too. In the future we should be able to turn this global variable into a local one.